Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:1.65.0-buster
```

Comment:
The commit message for this commit should be much more verbose. We can clean that up later.